tests/assertions/is_substring_of.zsh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env zunit
+
```

Comment:
Hmm. Interesting case. Actually if you swap the two values round, and run `[[ "some string" = *''* ]]` it passes. Not sure how best to deal with it.